src/test/resources/suite-testng1.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="SuitaTestNG" verbose="1" >
+    <test name="AltTest1" >
+        <classes>
+            <class name="sorinfoca.altpachet.AltTest" />
```

Comment:
If you remove the AltTest class be aware to remove this first <test> section from here too. Thanks!